atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTimestampsAndSentinels.java

```diff
@@ -23,13 +23,13 @@
 
 @Value.Immutable
 public abstract class SweepTimestampsAndSentinels {
-    public abstract Set<Long> timestamps();
-    public abstract Set<Cell> sentinelsToAdd();
-
     public static SweepTimestampsAndSentinels of(Set<Long> timestamps, Set<Cell> sentinelsToAdd) {
         return ImmutableSweepTimestampsAndSentinels.builder()
                 .timestamps(timestamps)
                 .sentinelsToAdd(sentinelsToAdd)
                 .build();
     }
+
+    public abstract Set<Long> timestamps();
+    public abstract Set<Cell> sentinelsToAdd();
 }
```

Comment:
constructors are usually at the top of files. Having them always in the same place tends to make the files much easier to skim through